libs/state/spec/rx-state.service.spec.ts

```diff
@@ -139,25 +139,33 @@ describe('RxStateService', () => {
     it('should return undefined as initial value', () => {
```

Comment:
can u also add dedicated tests for the `readOnly` that check if an error is thrown when trying to write to the readonly state? The apis shouldnt be accessible anymore